Doc/library/idle.rst

```diff
@@ -341,11 +341,18 @@ Set Breakpoint
 Clear Breakpoint
    Clear the breakpoint on that line.
 
-Shell and Output windows have the following.
+Shell and Output windows also have the following.
 
 Go to file/line
    Same as in Debug menu.
 
+The Shell window also has an output squeezing facility explained in the
+the *Python Shell window* subsection below.
+
+Squeeze
+   If the cursor is over an output line, squeeze all the output between
+   the code above and the prompt below down to a 'Squeezed text' button.
```

Comment:
> a 'Squeezed text' button

While it's implemented as a button, it doesn't quite behave like one, so this may be confusing to users. Perhaps "a 'Squeezed text' label"?

(later in the doc as well)